index.html

```diff
@@ -27,16 +27,25 @@ <h2 class="cta">Find events near you</h2>
       </div>
     </div>
   </div>
-  <div class="map">
-    <h2>Imagine a map</h2>
-  </div>
+  <div class="map" id="map"></div>
 
   <!-- FIXME bundle third party JS dependencies and serve them ourselves. 
        For now, just put them in here.  -->
   <script src="https://api.mapbox.com/mapbox.js/v3.0.1/mapbox.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.1/moment.min.js"></script>
   <!-- etc -->
 
-  <script src="./js/index.js"></script>
+  <!-- First draw map -->
```

Comment:
The events and zip code data are required dependencies to render the map. But for faster UX I split it up to render the map itself, first, then continue loading stuff. After adding webpack for bundling the details of how this works will need to change a bit.